tests/lib/nested.sh

```diff
@@ -24,15 +24,35 @@ NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL="${NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS
 
 nested_wait_for_ssh() {
     # TODO:UC20: the retry count should be lowered to something more reasonable.
-    nested_retry_until_success 800 1 "true"
+    local retry=800
```

Comment:
same comment as for `nested_wait_for_no_ssh`, can't we now use retry on the host system?

Reply:
ah wait but I suppose at this point we might not have `retry` on the system is that correct?

Reply:
I need to think how to use this using retry command, but I think it is possible. It should be included in a following pr

Reply:
I'll create a separate pr for this one today

Reply:
also when you change this in a different PR, can you decrease the retry count to something shorter like 400 maybe ?